src/api/spec/models/project_spec.rb

```diff
@@ -250,7 +255,7 @@
 
       it 'has ::' do
         property_of do
-          string = sized(1) { string(/[a-zA-Z0-9]/) } + sized(range(1, 199)) { string(/[-+\w\.]/) }
+          string = sized(1) { string(/[a-zA-Z0-9\-+\.:]/) } + sized(range(1, 199)) { string(/[-+\w\.:]/) }
```

Comment:
I would put `string(/[a-zA-Z0-9\-+]/)` instead of `string(/[a-zA-Z0-9\-+\.:]/)`. Fine for adding `-` and `+`, but no need to add `.` and `:` for the first character. There are two other places where this applies.

Reply:
Damn, you are right:sweat:

Reply:
![image](https://user-images.githubusercontent.com/3799140/44083905-8c545e5e-9fb5-11e8-8f8d-dbf7d019c9de.png)

Reply:
Nice catch @eduardoj